src/modules/pages/HealthCheckPage/UnhealthyReport.js

```diff
@@ -0,0 +1,54 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Link,
+} from 'react-router-dom';
+
+import block from '../../../helpers/bem-cn';
+
+import Icon from '../../Icon/Icon';
+
+const propTypes = {
+    className: PropTypes.string,
+};
+
+// temporary inline styles untill designer will provide warning icon
+const style = {
+    color: 'red',
+    fontSize: '30px',
+};
+
+class UnhealthyReport extends PureComponent {
+    state = {
```

Comment:
why not to keep the state in the store?

Reply:
at this point, I do not see the necessity of keeping this in store, which seems as overhead.